task-93/task-93.html

```diff
@@ -17,6 +17,7 @@
             <li>Элемент 4</li>
             <li>Элемент 5</li>
             <li>Элемент 6</li>
+            
         </ul>
```

Comment:
For the elements with `id="leftTable"` and `id="rightTable"` I'd strongly suggest you to use html element `<select>` with attribute `multiple` http://htmlbook.ru/html/select since it's logically and semantically more appropriate.
All the rest looks ok.